auto.js

```diff
@@ -478,6 +478,10 @@ var CookieAuto = {};
 
                 CookieAuto.updateGoalValue();
 
+                if (Game.onMenu !== "" && CookieAuto.ui.showing) {
+                    CookieAuto.ui.showMenu.call(CookieAuto.ui);
```

Comment:
why the complicated syntax?  Shouldn't this just be CookieAuto.ui.showMenu? 

Better yet, I'm not sure why ui was actually put inside CookieAuto.

Reply:
done